test/WebJobs.Script.Tests/ContentEndToEndTests.cs

```diff
@@ -372,22 +370,32 @@ protected Task Raw<Req>(Req content)
         }
     }
 
+    public class NodeContentFixture : ContentFixture, IDisposable
+    {
+        public NodeContentFixture() : base("Node")
+        {
+        }
+    }
+
     public class ContentFixture : IDisposable
     {
-        private readonly ScriptSettingsManager _settingsManager;
+        public readonly ScriptSettingsManager SettingsManager;
 
-        public ContentFixture()
+        public ContentFixture(string language)
         {
-            _settingsManager = ScriptSettingsManager.Instance;
+            SettingsManager = ScriptSettingsManager.Instance;
             FixtureId = "content";
 
             TraceWriter = new TestTraceWriter(TraceLevel.Verbose);
 
             ApiHubTestHelper.SetDefaultConnectionFactory();
 
+            RequestConfiguration = new HttpConfiguration();
+            RequestConfiguration.Formatters.Add(new PlaintextMediaTypeFormatter());
```

Comment:
Yep, I think I should be able to use the fixture.  Was initially inheriting from the EndToEndTests class but it had some built in tests that I want to avoid